### ISSUE TITLE
 Increased proportion of HLA DR+ Leukocytes

### ISSUE THREAD

**Issue by jpgourdine** [obophenotype/human-phenotype-ontology#4870]
**Preferred term label:**
Increased proportion of HLA DR+ Leukocytes

**Synonyms**
Increase level of HLA DR on Leukocytes
**Definition (free text, please give PubMed ID)**
An elevated proportion of leukocytes that express human leukocyte antigen (HLA)-DR.

**Parent term (use hpo.jax.org/app)**
Parent:
http://purl.obolibrary.org/obo/HP_0001881
Abnormal leukocyte morphology
HP:0001881

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

OMIM : 151430
OMIM: 601626
[sources LOINC](https://s.details.loinc.org/LOINC/32621-5.html?sections=Comprehensive
)

**Your nano-attribution (ORCID)**
0000-0001-9969-8610

**Comment by pnrobinson** [comment]
@jpgourdine I thought that leukocytes express HLA DR generally. I also cannot find any definition or cutoff in PubMed. The OMIM entries do not list this in the clinical synopsis. Can you please provide more context/information?

**Comment by nicolevasilevsky** [comment]
there is a LOINC code: 32621-5 HLA-DR [Presence]

Do you think we could annotate this to HP_0002852 Increased proportion of HLA DR+ T cells?

**Comment by pnrobinson** [comment]
Well to my knowledge HLA DR is normally present on T cells, and so I am not sure what this LOINC test is actually testing. I would suggest that we say cannot-annotate for this unless somebody has some idea of what this test is being used for.

**Comment by nicolevasilevsky** [comment]
ok - sounds good. We can close this then
